recipes/xhistogram/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "xhistogram" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7180e0366af54688562b5e8712ce60925aa4e565a7309c229402ba67fadd3a83
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python>=3.6
```

Comment:
This needs a single space `python >=3.6`

Reply:
Thanks!

Reply:
np!